respa_admin/static_src/styles/page-home.scss

```diff
@@ -210,11 +206,36 @@ main {
         }
       }
 
-      div.panel img, li div.empty-holder {
+      li div.empty-holder {
         position: absolute;
         width: $list-item-height;
         height: $list-item-height;
       }
+
+      div.panel img {
+          display: flex;
+          grid-area: 1.125;
+          width: $list-item-height;
+          height: $list-item-height;
+      }
+    }
+  }
+
+  section.resources {
+      @extend .main-section;
```

Comment:
Please change the indentation two 2 spaces here so it matches the file formatting